examples/games/game_menu.rs

```diff
@@ -379,16 +392,44 @@ mod menu {
         menu_state.set(MenuState::Main);
     }
 
-    fn main_menu_setup(mut commands: Commands, asset_server: Res<AssetServer>) {
-        // Common style for all buttons on the screen
-        let button_style = Style {
+    fn button_style() -> Style {
```

Comment:
This pattern was previously repeated verbatim in four different places. The only difference between the Main menu and the Main Settings menu was that the buttons were 50px wider on the Main menu. I've made the Main Settings menu buttons 50px wider in order to not have to repeat this configuration